string file
int8 row
int8 column
float32[] coords
---
int8 count
float32[] coords